filter-effects-2/Overview.bs

```diff
@@ -50,28 +53,283 @@ If the value of the 'backdrop-filter' property is ''backdrop-filter/none'' then
 filter effect applied. Otherwise, the list of functions are applied in the
 order provided.
 
-The first filter function or <a element>filter</a> reference in the list takes
-the element's BackgroundImage as the input image. Subsequent
+## Filtering and Clipping ## {#backdrop-filter-operation}
+
+An element (call it B) with a backdrop-filter property other than ''backdrop-filter/none'' is conceptually rendered as if
+the following steps are performed:
+1. Copy the <a>Backdrop Root Image</a> into a temporary buffer, such as a raster image. Call this buffer T’.
+2. Apply the backdrop-filter’s filter operations to the entire contents of T'.
+3. If element B has any transforms, apply the <b>inverse</b> of those transforms to the contents of T’.
```

Comment:
It's any transforms between B and the backdrop filter root, right?

Reply:
Good catch. Done.